index.html

```diff
@@ -29,8 +31,9 @@ <h1>Google</h1>
 			<input id="query" type="text" name="q" value="" placeholder="Search..."><br><br>
 
 			<div id="submit-button-row">
-				<button type="submit">Search</button>
+				<button type="submit" id="google-search-button">Google Search</button>
 				<span id="cursor"><i class="gg-arrow-top-left"></i></span>
```

Comment:
You make use of the cursor as space between the two buttons; Intentionally or unintentionally?
If you remove the cursor span, the buttons are directly attached to each other without any margin. But hey, it works.

Reply:
True, this use is unintentionally but it works.